src/Authenticator/AbstractAuthenticator.php

```diff
@@ -78,6 +85,32 @@ public function setIdentifier(IdentifierInterface $identifier)
         return $this;
     }
 
+    /**
+     * Get the last authentication result instance stored or null if none.
+     *
+     * @return ResultInterface|null
+     */
+    public function getLastResult()
```

Comment:
Does this need to go on the interface as well?

Reply:
yeah, I think it does.